modules.json

```diff
@@ -675,5 +678,16 @@
 			"totitle":"Template:UnsignedIP",
 			"format":"json"
 		}
+	},
+	{
+		"filename": "check_token",
+		"docstring": "Demo of `Checktoken` module: Get request to Check a CSRF token.",
```

Comment:
It seems that I can't see the change? The description is still the same

Reply:
@brendajerop , I've fixed it